_posts/2017-11-29-utdta-talk.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: tomesh @ UofT Decentralized Tech Association
+slug: decentralized tech talk
```

Comment:
slug url stuff should have dashes between.

Reply:
(and maybe match the title a little closer? i.e., add "talk" above or flip to "meetup".

Reply:
Good catch... should match the two exactly:

e.g.

```
title: "Install Party"
slug: install-party
```